FAQ.md

```diff
@@ -84,6 +85,18 @@ RuneLite is updated every couple weeks, usually immediately following the Wednes
 ## Where can I find the logs, screenshots, or configurations?
 To find logs, either open screenshot directory (by right-clicking "Camera" button) and navigate 1 directory up and then open logs folder, or navigate to `%userprofile%\.runelite\logs` on Windows or `$HOME/.runelite/logs` on Linux and macOS.
 
+## How can I change where screenshots get saved?
+It's not possible to directly change where RuneLite saves screenshots. However, you can create a symlink between the screenshots folder and where you actually want them saved.
+
+### On Windows:
+- Move your screenshots folder to where you want to have your screenshots saved, and then delete the `screenshots` folder that's in your `.runelite` folder.
```

Comment:
I assume this says move and delete (which technially can't be a thing; if it's moved, it doesn't exist anymore in the original location) since %userprofile%\.runelite\screenshots\ has to not exist anymore for symlinking to work?

Alternatively split it up into a second step that ensures the original path does not exist anymore.